lib/eslint-patch.js

```diff
@@ -1,33 +1,31 @@
-'use strict';
-var meld = require('meld');
-var docs = require('./docs');
-var Config = require("eslint/lib/config");
-var ConfigUpgrader = require('./config_upgrader');
-
-var supportedPlugins = ['react', 'babel'];
-
-module.exports = function patcher(eslint) {
-
-  meld.around(eslint.CLIEngine, 'loadPlugins', function(joinPoint) {
```

Comment:
@maxjacobson The function is only implemented on eslint 1.9 and lower, so my understanding is it wasn't doing anything.